doc/user_guide/annex_cfg.html

```diff
@@ -521,6 +521,23 @@ <h2 id="make">FCM Make Configuration</h2>
 </pre>
     </dd>
 
+    <dt id="make.name">name</dt>
+
+    <dd>
+      <p><dfn>description</dfn>: Specifies the NAME of the make, so that the
+      command will search for <q>fcm-make<var>NAME</var>.cfg</q> instead of
+      <q>fcm-make.cfg</q>, and will write context files in
+      <q>.fcm-make<var>NAME</var>/</q> and log files as
+      <q>fcm-make<var>NAME</var>.log</q>, etc.</p>
+
+      <p><dfn>value</dfn>: A file name friendly string.</p>
+
+      <p><dfn>example</dfn>:</p>
+      <pre>
+name = -friend
```

Comment:
I thought about that as well, but this is to make it easier to support `fcm_make2` in Rose.